<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auto_smzdmClass</class>
 <widget class="QMainWindow" name="auto_smzdmClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>阅读量</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>URL</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_thread_count">
           <property name="toolTip">
            <string>同时工作线程数</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox_url_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>9</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_add_url">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_remove_url">
           <property name="text">
            <string>移除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>comboBox_url_list</zorder>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_start">
          <property name="text">
           <string>开始连续</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_pause">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>354</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_import_URLs"/>
   <addaction name="action_clear_URLs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_import_URLs">
   <property name="text">
    <string>导入URL</string>
   </property>
   <property name="toolTip">
    <string>导入URL</string>
   </property>
  </action>
  <action name="action_clear_URLs">
   <property name="text">
    <string>清空URL</string>
   </property>
   <property name="toolTip">
    <string>清空URL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="auto_smzdm.qrc"/>
 </resources>
 <connections/>
</ui>
